eox_tenant/tenant_wise/proxies.py

```diff
@@ -150,20 +150,21 @@ def __get_value_for_org(cls, org, val_name, default=None):
         TenantConfig or Microsite model.
         """
         cache_key = "org-value-{}-{}".format(org, val_name)
-        if default is not None:
-            cache_key = "{}-{}".format(cache_key, default)
+        tenat_key = getattr(settings, "EDNX_TENANT_KEY", None)
```

Comment:
Can you please add a comment in explaining in detail as to why tenant key was needed here.